GVRf/Framework/framework/src/main/jni/engine/renderer/gl_renderer.cpp

```diff
@@ -414,6 +419,11 @@ namespace gvr
             GL(glEnable(GL_DEPTH_TEST));
         }
 
+        if (!render_data->depth_mask())
```

Comment:
If depth mask is not set in RenderData, enable render mask? something wrong here.